app/src/main/res/values/styles.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <style name="AppTheme" parent="Theme.AppCompat.Light.DarkActionBar">
+        <!-- Customize your theme here. -->
+        <item name="colorPrimary">@color/colorPrimary</item>
+        <item name="colorPrimaryDark">@color/colorPrimaryDark</item>
+        <item name="colorAccent">@color/colorAccent</item>
+    </style>
+
+    <style name="familyPlanningModu">
+        <item name="android:fontFamily">@font/roboto_medium</item>
+        <item name="android:textColor">#ffffff</item>
```

Comment:
Can you move these text-colors and text-color-hints to the `colors.xml` files and reuse them